### ISSUE TITLE
Link @ExternalPopulates and @ExternalRecollects of primitive to properties in the views

### ISSUE THREAD

**Issue by smaugho** [smaugho/declex#143]
When implementing MVVM setting (or reading) the values of properties (let's say Text, or Visibility), right now a method should be created, for instance:

```java
public int getViewIdVisibility() {
    //return visibility of component with id "viewId"
}
```

This is fine, and many times for processing the property it is required some function, but most of the time it would be great if you can just set this visibility in your program, at any moment, and at the end update the state of the view, it would be good to implement primitive fields which permits to do this, for instance:

```java
@Populate
int viewIdVisibility;
```

Then the "visibility" variable can be changed locally, and only when it is updated, the view will be informed of it (like calling $Populate(viewIdVisiblity), or calling $Populate(this).

This will permit faster implementations, specially in MVVM pattern where the View should be completely decoupled from the model. 

Basically, a method getViewIdVisiblity(), and setViewIdVisiblity() (in the case of a @Recollect) would be created to handle the interaction with the layout (similar to when a method is created).
